commercial-partner.md

```diff
@@ -0,0 +1,52 @@
+# Commercial Partner
+
+**Note:** This document is a work in progress
+
+The Karmada commercial partner program offers Karmada support, consulting, professional services for organizations embarking on their Karmada journey.
+
+Karmada commercial partners are any organization that successfully leverages the Karmada project for their service offering. Such as:
+- Using Karmada to build service on a public or private cloud.
+- Repackaging Karmada as a core component of a service offering.
+- Offering Karmada technical support, consultation services, and training.
+- Others leverage Karmada in the manner it was intended to serve their customers.
+
+### Why become a partner?
+
+If your company provides professional services to support Karmada deployments, please consider applying to become a Karmada commercial partner.
+
+The main benefits of becoming a Karmada commercial partner are:
+* Placement on the first tab of https://karmada.io/
```

Comment:
It's not clear to me, where is the first tab?

Reply:
nice finding. It should be the first tab of `https://karmada.io/partners`.

By the way, the page will be ready after https://github.com/karmada-io/website/pull/531, but the URL should be right.

Reply:
👍